docs/walk-through/the-structure-of-workflow-specs.md

```diff
@@ -1,18 +1,18 @@
 # The Structure of Workflow Specs
 
-We now know enough about the basic components of a workflow spec to review its basic structure:
+We now know enough about the basic components of a workflow spec. To review its basic structure:
 
 - Kubernetes header including meta-data
 - Spec body
-  - Entrypoint invocation with optionally arguments
-  - List of template definitions
+  * Entrypoint invocation with optional arguments
+  * List of template definitions
 
 - For each template definition
-  - Name of the template
-  - Optionally a list of inputs
-  - Optionally a list of outputs
-  - Container invocation (leaf template) or a list of steps
-    - For each step, a template invocation
+  * Name of the template
+  * Optionally a list of inputs
+  * Optionally a list of outputs
+  * Container invocation (leaf template) or a list of steps
+    + For each step, a template invocation
 
```

Comment:
hopefully, this will cause nested bulleting (referencing mkdocs documentation for unordered lists [here](https://squidfunk.github.io/mkdocs-material/reference/lists/#configuration)). Currently, the bullets aren't showing up as nested.

Reply:
Unfortunately, I needed to adjust the Makefile to skip this file for markdownlint, since that prefers 2 spaces (but "mkdocs build" needs 4 for nesting)